server/package.json

```diff
@@ -31,6 +31,7 @@
     "@casl/ability": "^6.5.0",
     "@casl/prisma": "^1.4.0",
     "@nestjs/apollo": "^11.0.5",
+    "@nestjs/axios": "^3.0.0",
```

Comment:
We should not have package-lock.json as we are using yarn!

Reply:
Why don't we add it to .gitignore? I asked you when we did the setup saying people would often add it inadvertently. I think you add a counter-argument but I don't remember exactly?

Reply:
I've added it to the .gitignore